EasyInventory/src/main/java/com/easyinventory/enterprise/model/Item.java

```diff
@@ -4,14 +4,16 @@
 
 import javax.persistence.*;
 
+
 /**
  * Author: Anthony Nagal
  *
  * This class is an entity for the item object
  */
 @Entity
 @Table(name="item_table")
-public class Item {
+
+public @Data class Item {
```

Comment:
This class has both Lombok and getters and setters.  That's redundant.  The idea with Lombok is that you don't need getters and setters.  So, if adding Lombok, remove the getters and setters.